src/components/App/Show/index.js

```diff
@@ -0,0 +1,54 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import { fetchShow } from 'redux/actions/showsActions';
+
+import styles from './Show.module.scss';
+
+class Show extends Component {
+  componentDidMount() {
+    const { id } = this.props.match.params;
+    this.props.fetchShow(id);
+  }
+
+  render() {
+    const { name, image, summary, runtime, status, type, schedule } = this.props.show;
+    const { time } = schedule || '';
+    const days  = schedule ? schedule.days.join(', ') : '';
+
+    if (!image) {
+      return <p>Loading...</p>;
+    }
+
+    return (
+      <>
+        <figure className={styles.showInfo}>
+          <img src={image.original} alt={name} className={styles.showImage}/>
+            <figcaption className={styles.description}>
+              <h3>{name}</h3>
+              <p className={styles.summary}>{summary.replace(/(<([^>]+)>)/ig,"")}</p>
+              {/* the above RegExp strips summary from HTML tags parsed as plain text */}
+              <p className={styles.time}><strong>WATCH IT AT: </strong>{time} on {days}</p>
+              <p className={styles.details}><strong>RUNTIME: </strong>{runtime} min.</p>
+              <p className={styles.details}><strong>STATUS: </strong>{status}</p>
+              <p className={styles.details}><strong>TYPE: </strong>{type}</p>
+            </figcaption>
+        </figure>
+        <Link to="/page/1" className={styles.backHome}>Back to main page</Link> 
```

Comment:
Are you sure that when user was on 3rd page and clicked to see the Show wants to go back to the first page? And then go to the third again?

Reply:
You're right. I've change it to a "real" send back feature. I just wonder if I can somehow handle the case when user enters the app directly into a given show's page. Because then, when he clicks "back", he's obviously taken away from the app altogether. Is this fine?